core/GlobalState.cc

```diff
@@ -607,6 +634,38 @@ SymbolRef GlobalState::lookupSymbolWithFlags(SymbolRef owner, NameRef name, u4 f
     return Symbols::noSymbol();
 }
 
+SymbolRef GlobalState::findRenamedSymbol(SymbolRef owner, SymbolRef sym) {
```

Comment:
This method works by knowing how to replicate the logic of renaming in order to find whatever the previous name was: for `x$n` where `n` is larger than 2, it'll be `x$(n-1)`, for bare `x`, it'll be whatever the largest `x$n` that exists is, if any; otherwise, there will be none.

Reply:
can we leave this comment in source?